### DISCUSSION TITLE
An issue in installation

### DISCUSSION THREAD

**Discussion by Yiqing-Gu** [USC-ACTLab/crazyswarm#459] [Q&A] [answered]
Hello everyone, 

Working surrounding is Ubuntu 20.04 in VM. The version of python is 3.9.6. I chose "simulation only" in crazyswarm documentation. Although I did installation step by step, I got stuck on running ./buildSimOnly.sh.

(crazyswarm) ubuntu@ubuntu:~/Desktop/crazyswarm$ ./buildSimOnly.sh
Cloning into '/home/ubuntu/Desktop/crazyswarm/crazyflie-firmware'...

I saw my computer kept cloning and nothing happened next. System did not die. I checked "crazyflie-firmware" folder but there was nothing in it.

I am not sure what's wrong here. Could you guys help me figure out?

**Comment by whoenig** [comment, marked as answer]
This part just uses git to checkout the firmware. If that causes problems, then either your Internet connection does not work properly in the VM or the (github) server was not reachable when you tried.

**Reply by Yiqing-Gu** [reply]
Thanks a lot. I tried again again and it worked. However, after running "$CSW_PYTHON -m pytest", I got one test failed. The following is the error information:

test_videoOutput.py::test_videoOutput
  /home/ubuntu/anaconda3/envs/crazyswarm/lib/python3.9/site-packages/past/builtins/misc.py:45: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    from imp import reload

Does that mean I need to revise a script in the folder?

**Reply by whoenig** [reply]
That's just a warning, not an error.
